admin-dev/themes/new-theme/js/app/widgets/ps-number.vue

```diff
@@ -85,7 +85,7 @@
         this.$emit('blur', $event);
       },
       increment() {
-        const value = parseInt(this.value === '' ? 0 : this.value, 10);
+        const value = parseInt(this.value === '' || isNaN(this.value) ? 0 : this.value, 10);
```

Comment:
Then the vue prop type of `value` should not be `Number` but `String` no? As every inputs values are strings